README.md

```diff
@@ -1,4 +1,16 @@
 wp-cli/ideas
 ============
 
-Have an idea for a WP-CLI command? [Create a new issue](https://github.com/wp-cli/ideas/issues/new) or [up-vote an existing issue](https://github.com/wp-cli/ideas/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc).
+### Have an idea for a WP-CLI command?**
+
+Feel free to [up-vote an existing issue](https://github.com/wp-cli/ideas/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc) if you can find one that matches, or [create a new issue](https://github.com/wp-cli/ideas/issues/new) if no one has proposed the idea yet.
```

Comment:
What does "up-voting" actually mean? If we provide a strict procedure, like add a 👍  for up-voting and a 👎 for down-voting, on the initial post of an issue (the first one), we can use this "pseudo-structured" information to build a dashboard with the best ideas.
This dashboard can then serve as the main entry-point for our feature implementation process, showing an overview of our "ideas-roadmap", and providing links to work on the existing one or create a new one.